xilem/src/lib.rs

```diff
@@ -1,14 +1,16 @@
 // Copyright 2024 the Xilem Authors
 // SPDX-License-Identifier: Apache-2.0
 
-#![cfg_attr(docsrs, feature(doc_auto_cfg))]
-#![deny(clippy::trivially_copy_pass_by_ref)]
-// LINEBENDER LINT SET - v1
+//! An experimental Rust native UI framework.
```

Comment:
Oops :sweat_smile: (I should have written something here before)

Reply:
I just copy-pasted the GitHub description field. Feel free to update both. 🚀